include/taskolib/Tag.h

```diff
@@ -106,6 +106,18 @@ class Tag
         return a.name_ >= b.name_;
     }
 
+    /// Concatenate a string and a tag.
+    friend std::string operator+(const std::string& lhs, const Tag& rhs)
```

Comment:
According to the [`gul14::join`](https://gul14.info/group__join__split__h.html#ga0b05ef3db1b337691307dada4bb755aa) it should handle the `operator+=` (assignment operator) but here you use the `operator+` (arithmetic operator). I would suspect that using a free function solves all relevant use cases ...?

Reply:
Dang it... you are right, I did not read the docs correctly. Thanks!

I'll add an `operator+=`.

Reply:
Done, now it is there. 🤦